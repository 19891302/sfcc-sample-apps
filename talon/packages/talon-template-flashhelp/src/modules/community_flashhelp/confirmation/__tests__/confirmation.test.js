import { createElement } from 'lwc';
import Confirmation from 'community_flashhelp/confirmation';

describe('community_flashhelp/confirmation', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-confirmation', { is: Confirmation });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
