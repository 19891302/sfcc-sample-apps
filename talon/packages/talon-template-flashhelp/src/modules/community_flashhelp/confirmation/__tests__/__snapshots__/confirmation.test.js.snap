// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/confirmation renders correctly 1`] = `
<community_flashhelp-confirmation>
  #shadow-root(open)
    <div
      class="confirmation"
    >
      <div
        class="container-fluid slds-text-align_center"
      >
        <h1
          class="slds-text-heading_large"
        >
          Thank you for applying for the Hurricane Harvey Immediate Assistance Program. You will receive an email/text in the next 24 hours regarding the status of your application.
        </h1>
      </div>
    </div>
</community_flashhelp-confirmation>
`;
